verticapy/core/vdataframe/_math.py

```diff
@@ -43,20 +43,20 @@ class vDFMath(vDFFilter):
     def __abs__(self) -> "vDataFrame":
         return self.copy().abs()
 
-    def __ceil__(self, n: int) -> "vDataFrame":
+    def __ceil__(self) -> "vDataFrame":
         vdf = self.copy()
         columns = vdf.numcol()
         for col in columns:
             if vdf[col].category() == "float":
-                vdf[col].apply_fun(func="ceil", x=n)
+                vdf[col].apply_fun(func="ceil")
         return vdf
 
-    def __floor__(self, n: int) -> "vDataFrame":
```

Comment:
why those changes?

Reply:
These are called dunder or magic methods (https://www.geeksforgeeks.org/dunder-magic-methods-python/).
It is convention that when an object implements __ceil__() function it actually implements a behavior for math.ceil(obj). That function doesn't accept any extra argument. Besides, what is the use of a precision argument for ceil and floor functions?
https://blog.finxter.com/python-__ceil__-magic-method
https://mathspp.com/blog/pydonts/dunder-methods